_layouts/doc.html

```diff
@@ -7,10 +7,21 @@
     <div class="row">
       <ol class="breadcrumb">
         <li><a href="{{site.baseurl}}/">Home</a></li>
-        {% for bc in page.breadcrumbs %}
-        <li><a href="{{site.baseurl}}/{{bc}}">{{bc}}</a></li>
+        {% assign splitted_url = page.url | split:"/" %}
+        {% assign repo_name = splitted_url[2] %}
+        {% for repo in site.docs_repos %}
+          {% if repo.name == repo_name %}
+            {% assign description = repo.description %}
+          {% endif %}
         {% endfor %}
-        <li class="active">{{ page.title }}</li>
+        {% if splitted_url.size > 3 %}
```

Comment:
@apojomovsky if this is checking for the main page (thanks for the offline explanation :)), consider a more robust approach (e.g. checking if `page.url` ends with `repo_name`).

Reply:
Hm, it seems that there isn't a `liquid` native operation that implements an "ends with" like query. (See [here](https://stackoverflow.com/a/37026674)). What about doing:
```
{% if repo_name != page_name %}
```
Which will be equal if the `URL` corresponds to the root of the package.